website/docusaurus.config.ts

```diff
@@ -218,6 +218,14 @@ const config: Config = {
       theme: prismThemes.vsDark,
       darkTheme: prismThemes.vsDark,
     },
+    announcementBar: {
+      id: "survey_bar",
+      content:
+        'Help us improve MLflow by taking our <a target="_blank" rel="noopener noreferrer" href="https://surveys.training.databricks.com/jfe/form/SV_cA2jrfBjs6vi6SG">survey</a>!',
+      backgroundColor: "#72A1ED",
+      textColor: "#091E42",
```

Comment:
```suggestion
      backgroundColor: "#0194e2",
      textColor: "#ffffff",
```

<img width="1245" alt="Screenshot 2024-12-20 at 10 01 00 AM" src="https://github.com/user-attachments/assets/10d5ade9-ac50-445b-91f6-953d53de2426" />

Reply:
A most excellent change :D